src/presenter/createQuestionPresenter.js

```diff
@@ -0,0 +1,34 @@
+import model from "../model/model";
+
+var index = 1;
```

Comment:
For sure this index should have been in the model. Also, it works from value `1` just because you have an initial question with ID = `-1` instead of `2`.